src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/DraftApprenticeshipControllerTests/AddDraftApprenticeshipTests.cs

```diff
@@ -128,8 +128,6 @@ public AddDraftApprenticeshipTestsFixture()
             AuthorizationService = new Mock<IAuthorizationService>();
             EncodingService = new Mock<IEncodingService>();
 
-            AuthorizationService.Setup(x => x.IsAuthorized(EmployerFeature.EnhancedApproval)).Returns(false);
```

Comment:
Also a using statement at the top of this class which can be removed

Reply:
this has been updated